app/policies/google_policy.rb

```diff
@@ -0,0 +1,13 @@
+class GooglePolicy < BasePolicy
+  def email
+    nil
+  end
+
+  def oauth_expires
+    Time.at auth_data.credentials.expires_at
```

Comment:
Do not use `Time.at` without zone. Use one of `Time.zone.at`, `Time.at.current`, `Time.at.in_time_zone`, `Time.at.utc`, `Time.at.getlocal`, `Time.at.iso8601`, `Time.at.jisx0301`, `Time.at.rfc3339`, `Time.at.to_i`, `Time.at.to_f` instead.